test/test_parse_en.py

```diff
@@ -687,6 +687,11 @@ def testExtract(text, expected_date, expected_leftover):
         testExtract("what's the weather like next tuesday night",
                     "2017-07-04 22:00:00", "what is weather like night")
 
+    def test_extract_date_years(self):
+        date = datetime(2017, 6, 27, tzinfo=default_timezone())  # Tue June 27, 2017
+        self.assertEqual(extract_datetime('in 2007', date)[0],
+            datetime(2007, 6, 27, tzinfo=date.tzinfo))
```

Comment:
I think we should bring this up with users, because there's definitely no correct answer, but this seems jarring to me

Reply:
it makes sense if you think deeper about the meaning of anchorDate, but at first it is very jarring at first indeed

regardless im calling this out of scope, this is a quick bugfix only. Keep in mind the full rewrite is still happening so this is a stop gap and i don't want to devote a lot of time to it

Reply:
Yeah it seems like we really want a "granularity" indicator here too - but this is better than what exists.